src/rgw/driver/sfs/types.cc

```diff
@@ -258,52 +323,75 @@ void MultipartUpload::abort(const DoutPrefixProvider *dpp) {
   objref.reset();
 }
 
-bool Bucket::want_new_version(const rgw_obj_key& key, ObjectRef obj) {
-  return !(key.instance.empty() || key.instance == obj->instance);
+bool Bucket::want_specific_version(const rgw_obj_key& key) {
```

Comment:
the comment on the previous commit goes for this one, I would rather have `s/want/wants/`.

Reply:
:+1: agreed. Replacing with local variable as well, the method doesn't make much sense outside get_or_create